File: GC.Domain/Gardens/Appeals/AppealBlank.cs
using System;

namespace GC.Domain.Gardens.Appeals
{
    public class AppealBlank
    {
        public Guid? Id { get; set; }
        public String FirstName { get; set; }
        public String LastName { get; set; }
        public String PhoneNumber { get; set; }
        public String Title { get; set; }
        public String Message { get; set; }
        public String Email { get; set; }
        public DateTime? Date { get; set; }
    }
}

File: GC.WebSpace/ClientApp/domain/gardens/appeals/appealBlank.ts
export class AppealBlank {
    constructor(
        public id: string | null,
        public firstName: string | null,
        public lastName: string | null,
        public phoneNumber: string | null,
        public title: string | null,
        public message: string | null,
        public email: string | null,
        public date: Date | null,
    ) { }

    public static empty = () => {
        return new AppealBlank(null, null, null, null, null, null, null, null);
    }
}